ferrocene/doc/safety-manual/src/self-test/constraints.rst

```diff
@@ -0,0 +1,32 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Constraints
+===========
+
+.. constraint:: Do not modify toolchain
+   :id: CONSTR_SELF_TEST_DO_NOT_MODIFY
+   :mitigates: ERR_SELF_TEST_FALSE_NEGATIVES, ERR_SELF_TEST_FALSE_POSITIVES
+
+   While executing the tool,
+   the user must ensure that the toolchain is not modified.
```

Comment:
How do we expect the users to do this?

Reply:
bad wording... I meant that they should not modify the toolchain while self-test is running

Reply:
Sure, but if it's in a constraint, how are users supposed to adhere to this constraint?

Reply:
5be9f72f097

Reply:
the users should avoid modifying the toolchain

Reply:
Oh I see what you mean, I interpreted this as "the user must ensure the self-testing tool did not modify the toolchain on its own".

I think the new wording already improves the situation there, but I'd maybe change it to:

> The user must not modify the toolchain while the self-testing tool is being executed.

Reply:
218bb2d345a